app/views/events/_transfer_form.html.erb

```diff
@@ -6,30 +6,41 @@
     data-transfer-form-wire-value="<%= new_event_wire_path(@event) %>">
     <div data-transfer-form-target="home">
       <fieldset>
-        <legend class="heading h2 pt2 pb2">What type of transfer?</legend>
+        <legend class="heading h2 pb2">What type of transfer?</legend>
         <div class="field field--options fit">
           <%= link_to new_event_ach_transfer_path(@event), class: "text-decoration-none btn--form-option", data: { turbo_frame: "_top" } do %>
             <%= inline_icon "payment-transfer", size: 28 %>
             <strong>ACH transfer</strong>
+            <small>United States</small>
           <% end %>
           <%= link_to new_event_increase_check_path(@event), class: "text-decoration-none btn--form-option", data: { turbo_frame: "_top" } do %>
             <%= inline_icon "email", size: 28 %>
             <strong>Mailed check</strong>
+            <small>United States</small>
           <% end %>
-          <%= link_to new_event_wire_path(@event), class: "text-decoration-none btn--form-option", data: { turbo_frame: "_top" } do %>
+          <% if Flipper.enabled?(:wise_transfers_2025_07_31, @user) %>
+            <%= link_to new_event_wise_transfer_path(@event), class: "text-decoration-none btn--form-option", data: { turbo_frame: "_top" } do %>
+              <%= inline_icon "wise", size: 28 %>
+              <strong>Wise transfer</strong>
+              <small>International</small>
+            <% end %>
+          <% end %>
+          <%= link_to new_event_wire_path(@event), class: "text-decoration-none btn--form-option relative overflow-hidden", data: { turbo_frame: "_top" } do %>
             <%= inline_icon "web", size: 28 %>
-            <strong>International wire</strong>
+            <div class="badge badge--corner bg-info m1 mt0" style="width: fit-content;">$500 minimum</div>
+            <strong>Wire transfer</strong>
+            <small>International</small>
           <% end %>
           <%= link_to new_disbursement_path(source_event_id: @event), class: "text-decoration-none btn--form-option", data: { turbo_frame: "_top" } do %>
             <%= inline_icon "bank-account", size: 28 %>
             <strong>HCB transfer</strong>
-            <small>(between HCB organizations)</small>
+            <small>Any HCB organization</small>
```

Comment:
I think the original label was clearer

Reply:
To another HCB organization - though I'm not sure if we need a consistent label

Reply:
`Another HCB organization` sounds like it would work well?